edelphi/src/main/webapp/_scripts/gui/panel_admin/queryeditorblockcontroller.js

```diff
@@ -4295,6 +4311,24 @@ QueryEditorScale2DThesisPageEditor = Class.create(QueryEditorThesisPageEditor, {
   }
 });
 
+QueryEditorMultipleScale2DThesisPageEditor = Class.create(QueryEditorThesisPageEditor, {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add the "let", "const" or "var" keyword to this declaration of "QueryEditorMultipleScale2DThesisPageEditor" to make it explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS2703)